locales/nl/json.json

```diff
@@ -326,7 +326,7 @@
     "Finish enabling two factor authentication.": "Inschakelen van tweestapsverificatie afronden.",
     "Finland": "Finland",
     "Florida": "Florida",
-    "For security reasons your new card will be charged a minimal fee upon registration.": "Om veiligheidsredenen wordt uw nieuwe kaart bij registratie een minimale vergoeding in rekening gebracht.",
+    "For security reasons your new card will be charged a minimal fee upon registration.": "Om veiligheidsredenen word je nieuwe kaart bij registratie een minimale vergoeding in rekening gebracht.",
```

Comment:
This one is incorrect. It should be `wordt` because the subject is `je nieuwe kaart`

Reply:
Besides that the sentence isn't right, this one is better I guess:
`"Om veiligheidsredenen wordt bij registratie van je nieuwe kaart een minimale vergoeding in rekening gebracht.`

Reply:
Oops, you're absolutely right. And nice suggestion, I will commit the change.